src/components/pages/BookPage/starRate.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+function CreateStars({ rate }) {
+  let stars = [];
+  let firstDigit = rate - parseInt(rate);
+  let secondDigit = firstDigit.toString().split('.')[1];
+  let createStars = () => {
+    for (let i = 1; i <= 5; i++) {
+      if (i <= rate) {
+        stars.push(<icon className="fa fa-star text-yellow-300 mr-1 "></icon>);
+      } else {
+        if (secondDigit == 5) {
```

Comment:
first of all you have to use three equal sign `===` always
second what dose this line mean? if the secondDigit is more or less than 5 you will count it as zero, right?

Reply:
this to get if there .5 to make half star ,and you just told me to make the half star